presto-main/src/main/java/com/facebook/presto/operator/aggregation/ArbitraryAggregationFunction.java

```diff
@@ -198,4 +210,47 @@ public static void input(Type type, BlockState state, Block block, int position)
         }
         state.setBlock((Block) type.getObject(block, position));
     }
+
+    public static void combine(NullableLongState state, NullableLongState otherState)
+    {
+        if (!state.isNull() || otherState.isNull()) {
```

Comment:
I would remove all `otherState.isNull()` checks here for the same reason I mentioned before.